### DISCUSSION TITLE
GitHub Copilot broken after switching to SSO with 2FA for organization?

### DISCUSSION THREAD

**Discussion by bravehager** [community/community#12887] [Copilot Conversations] [answered]
I recently switched to SSO with 2FA required for one of my GitHub organizations. Ever since then, I'm getting a "User not authorized" error when I try to enable Copilot in VS Code.

Here's the output from the Github Copilot extension:

```
[INFO] [auth] [2022-03-15T15:09:36.012Z] Invalid copilot token: missing token: 401 
[ERROR] [default] [2022-03-15T15:09:36.015Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Comment by davecheney** [comment, marked as answer]
I've confirmed that your invite is active and you have access to copilot. 

<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by Yinzy00** [comment]


**Reply by verdihatorou** [reply]
Yeah
